scenarios/segmentation/11_exploring_hyperparameters.ipynb

```diff
@@ -0,0 +1,1037 @@
+{
```

Comment:
<p>maybe it would be worth adding one more line after this graph to explain how to intepret it? Like that the column on the right side is has performed the best but has taken the longest and that is the trade off.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Agree. Added 1-2 sentences discussion of results.